bin/generate-manifest.js

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env node
+
+const path = require('path');
+const fs = require('fs');
+const packageMeta = require('../package.json');
+
+const manifest = {
```

Comment:
Well, I figured it would be easier to have it all in one place to edit for the things pulled from package.json and the other non-templated properties. Otherwise, it'll be split between a template file and a build script and possibly be confusing that way too